.circleci/build_win.ps1

```diff
@@ -2,22 +2,24 @@ $ErrorActionPreference = "Stop"
 
 cd "$PSScriptRoot\.."
 
+mkdir build
+cd build
+$boost_dir=(Resolve-Path $PSScriptRoot\..\deps\boost\lib\cmake\Boost-*)
+
 if ("$Env:FORCE_RELEASE" -Or "$Env:CIRCLE_TAG") {
-	New-Item prerelease.txt -type file
 	Write-Host "Building release version."
+	..\deps\cmake\bin\cmake --build . -j 10 --target install --config Release -DSOL_RELEASE_VERSION="On"
```

Comment:
The cmake options here and below don't seem to match the new cmake options defined below, do they?
Also it doesn't seem nice to invoke ``cmake`` here and then again below...